charts/elasti/templates/resolver-additional-access-rbac.yaml

```
@@ -5,9 +5,6 @@ metadata:
   labels:
     {{- include "elasti.labels" . | nindent 4 }}
 rules:
-- apiGroups: ["apps"]
-  resources: ["deployments"]
-  verbs: ["get", "list", "watch", "update", "patch"]
-- apiGroups: [""]
-  resources: ["services", "pods", "endpoints"]
-  verbs: ["get", "list", "watch", "update", "patch", "delete", "create"]
\ No newline at end of file
+- apiGroups: ["discovery.k8s.io"]
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 0

(indentation)